backend/src/routes/__test__/encounter.route.test.ts

```diff
@@ -426,6 +429,436 @@ describe('PUT /encounters/:id ', () => {
     })
 });
 
+// Delete Encounters Endpoint tests
+
+// Delete Encounter 200
+describe('DELETE /encounter/:id', () => {
+    it('Successfully deletes single encounter with single person: ', async () => {
+        // Get Authentication ID for User
+        const auth_id = await testUtils.getAuthIdFromToken(token);
+
+        // Create Person
```

Comment:
There's a lot of repeated code between these test cases. I'd strongly recommend creating some functions to create people & encounters for your tests in order to remove a bunch of this repeated code at the start of each test case.

Reply:
conrad what are you doing here?